server/utils/formUtils.ts

```diff
@@ -18,4 +18,12 @@ export default class FormUtils {
 
     res.locals.errors = { list, messages }
   }
+
+  static setFormValues(req: Request, res: Response): void {
+    const formValuesAsString = req.flash('formValues')[0]
+
+    const formValues = formValuesAsString ? JSON.parse(formValuesAsString) : {}
+
+    res.locals.formValues = formValues
```

Comment:
Would you mind adding a commit to update the `course` view and the two actions that use it to use `formValues` instead of `values` to bring it in line with this?

Reply:
Have pushed another commit with this change.

Reply:
Thanks!